Phase-5/SearchWebAPI/Controllers/SearchController.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using Microsoft.AspNetCore.Mvc;
+using SearchEngine.Classes.IO.Database.Models;
+using SearchEngine.Interfaces.Core;
+
+namespace SearchWebAPI.Controllers
+{
+    [ApiController]
+    [Route("[controller]/[action]")]
+    public class SearchController : Controller
+    {
+        private readonly ISearchEngineCore _searchEngineCore;
+
+        public SearchController(ISearchEngineCore searchEngineCore)
+        {
+            _searchEngineCore = searchEngineCore;
+        }
+
+        [HttpGet()]
+        public List<int> Query(string toSearch)
```

Comment:
IDE give this fact about this HttpGet():
Parentheses are redundant if attribute has no arguments